src/publish.js

```diff
@@ -28,6 +28,35 @@ async function publishOutput() {
   const OUTPUT_BRANCH_SUFFIX = core.getInput("output-branch-suffix");
   if (GITHUB_EVENT_NAME.includes("pull_request")) {
     const PR_NUMBER = github.context.payload.pull_request.number;
+
+    // Build PDF diff
+    if (core.getInput("build-diff-on-pull-request") == "true") {
+      const LATEXDIFF_URL = core.getInput("latexdiff-url");
+      const LATEXPAND_URL = core.getInput("latexpand-url"); 
+      const BASE_REF = github.context.payload.pull_request.base.ref;
+
```

Comment:
Can we wrap this in a `try-catch` block? There are probably failure modes we haven't thought of, so it would be nice to throw a warning instead of having the entire workflow fail if the diff can't be generated.